<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractDetailsWidget</class>
 <widget class="QWidget" name="ContractDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_53">
     <property name="text">
      <string>Security Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="securityTypeComboBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="symbolLabel">
     <property name="text">
      <string>Symbol</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="symbolLineEdit">
     <property name="text">
      <string>MSFT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="localSymbolLabel">
     <property name="text">
      <string>Local Symbol</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="localSymbolLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="exchangeLabel">
     <property name="text">
      <string>Exchange</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="exchangeLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>SMART</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="currencyLabel">
     <property name="text">
      <string>Currency</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="currencyLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>USD</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="expiryLabel">
     <property name="text">
      <string>Expiry</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="expiryLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="strikeLabel">
     <property name="text">
      <string>Strike</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="strikeLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="rightLabel">
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="rightLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="multiplierLabel">
     <property name="text">
      <string>Multiplier</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="multiplierLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="comboLegsLabel">
     <property name="text">
      <string>Combo Legs</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="comboLegsLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="primaryExchangeLabel">
     <property name="text">
      <string>Primary Exch</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="primaryExchangeLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="contractIdLabel">
     <property name="text">
      <string>Contract Id</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="contractIdLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="tradingClassLabel">
     <property name="text">
      <string>Trading Class</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="tradingClassLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="includeExpiredLabel">
     <property name="text">
      <string>Include Expired</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="includeExpiredLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="secIdTypeLabel">
     <property name="text">
      <string>Sec Id Type</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="secIdTypeLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="securityIdLabel">
     <property name="text">
      <string>Security Id</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="securityIdLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="comboLegsDescLabel">
     <property name="text">
      <string>Combo Legs Desc</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="comboLegsDescLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="underCompLabel">
     <property name="text">
      <string>Under Comp</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLineEdit" name="underCompLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
